.codeclimate.yml

```diff
@@ -51,3 +51,8 @@ plugins:
     enabled: true
   sass-lint:
     enabled: true
+    rules:
+      nesting-depth:
+        - 2
+        -
+          max-depth: 5
```

Comment:
sass-lint is crazy in that it won't let same-line selectors, but also has a max nesting depth of 2, (so why use sass...)?
We need higher specificity especially to avoid `important` spam over primevue defaults sometimes.

And yes this is the documented config syntax https://github.com/sasstools/sass-lint#rules

Reply:
It should work that way, though I saw the sass-lint documentation and issues do it as above in multiple spots so wasn't sure if it's more or less confusing to match that documentation or not